tools/mo/openvino/tools/mo/back/offline_transformations.py

```diff
@@ -57,24 +100,14 @@ def read_model(path_to_xml):
 
     func = read_model(input_model + "_tmp.xml")
 
-    apply_user_transformations(func, transforms)
+    add_layouts(func, argv)  # TODO: replace with preprocessing
+
```

Comment:
something is incomplete here. I presume we should call preprocessing here with configured layout for each input